### DISCUSSION TITLE
Failed to execute LQI for 'Coordinator'

### DISCUSSION THREAD

**Discussion by audun-v** [Koenkk/zigbee2mqtt#17689] [Devices]
Hi 😃 

I am running Home Assistant on an Intel i7 computer.
I have just installed zigbee2mqtt in HomeAssistant with a Sonoff Zigbee 3.0 USB Dongle Plus-P.
The Sonoss Dongle is flashed with the latest firmware.

I have added a totalt of 12 devices  and everything seems to be working good.
The problem is if I go to the tab with the map and click update map. 
Then I get the error message "Failed to execute LQI for 'Coordinator'".
That 's the only device that get that error.

I have tried to reflash the firmware on the Sonoff dongle, changed usb bort, rebootet the computer...
From the log file:

`Error 2023-05-15 14:48:46Failed to execute LQI for 'Coordinator'`
`Debug 2023-05-15 14:48:46Error: AREQ - ZDO - mgmtLqiRsp after 10000ms at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35) at listOnTimeout (node:internal/timers:559:17) at processTimers (node:internal/timers:502:7)
`
I'm also wondering why no end devices connects directly to the coordinator? Is that normal behaviour, or could that be a part of the error message?

If someone can help with this I would be very happy!

![image](https://github.com/Koenkk/zigbee2mqtt/assets/56632644/7869a53f-f1c3-4938-96f3-1be056aa682e)

![image](https://github.com/Koenkk/zigbee2mqtt/assets/56632644/1bb2afcf-4cc8-4446-b41e-abc2f4737bbb)

![image](https://github.com/Koenkk/zigbee2mqtt/assets/56632644/e04a289d-4dc0-458c-9ec1-ecf1f11c4f10)

**Comment by audun-v** [comment]
I managed to fix this problem. Deleted all the config and started over. Everything ok!

**Comment by bentwookie** [comment]
I'm running into this same issue, but don't really want to start over. Any ideas on how to fix it?

**Comment by KevinGaudry** [comment]
Hi, I have the same issue and tried already evertything... any clue ?
